src/index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Document</title>
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+	<h1>Universal Declaration of Human Rights</h1>
+<h2>Preamble</h2>
```

Comment:
fixed whitespaces

Reply:
It's unreal to read this

Reply:
about whitespaces - they must be -but only two, or they don't must be at all?

Reply:
they must be